api.json

```diff
@@ -0,0 +1,761 @@
+            "thursday?": "Boolean",
+            "friday?": "Boolean",
+            "saturday?": "Boolean",
+            "sunday?": "Boolean"
+          }
+        }
+      },
+      "/:id/feedback": {
+        "GET": {
+          "description": "Retreive feedback for one organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "since": "String",
+            "until": "String"
+          },
+          "response": {
+            "-1": "Number",
+            "0": "Number",
+            "1": "Number",
+            "since": "String",
+            "until": "String"
+          }
+        },
+        "POST": {
+          "description": "Send feedback for one organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "feedbackScore": "Number"
+          },
+          "response": {
+            "feedbackScore": "Number",
+            "organisationId": "Number"
+
+          }
+        }
+      },
+      "/:id/users": {
+        "GET": {
+          "description": "Retreive all users in an organisation",
+          "auth": {},
+          "scope": {},
+          "query": {
+            "page": "Number",
+            "offset": "Number",
+            "sort": "String",
+            "gender": "String",
+            "age": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean"
+          },
+          "response": [{
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean"
+          }]
+        }
+      },
+      "/:id/visits": {
+        "GET": {
+          "description": "Retreive all visit data for an organisation",
+          "auth": true,
+          "scope": {},
+          "query": {
+            "fields": ["String"],
+            "sort": "String",
+            "page": "Number",
+            "offset": "Number",
+            "gender": "String",
+            "age": "String",
+            "activity": "String"
+          },
+          "response": {}
+        }
+      },
+      "/:id/volunteers": {
+        "GET": {
+          "description": "Retrieve volunteer details for a single organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "since": "String",
+              "until": "String"
+            }
+          },
+          "response": {
+            "totalVolunteers": "Number",
+            "totalVolunteerHours": "Number",
+            "volunteerLogs": [{
+              "volunteerLogId": "Number",
+              "userId": "Number",
+              "volunteerActivity": "String",
+              "duration": "Number",
+              "loggedAt": "String"
+            }]
+          }
+        }
+      },
+      "/:id/outreach": {
```

Comment:
Is this gonna have more nested routes? Like `/:id/outreach/:oid`, `/:id/outreach/:oid/meetings`? Or is that handled some other way?

Reply:
Outreach campaigns for organisations will be handled for in the org endpoint. This is just for outreach campaign target types.

.. although this is subject to change

Reply:
This isn't for target types, the response is for actual campaigns. There is a filter by type though.